packages/framework/src/Form/DatePickerType.php

```diff
@@ -15,9 +15,9 @@ class DatePickerType extends AbstractType
     public const FORMAT_JS = 'dd.mm.yy';
 
     /**
-     * @param \Shopsys\FrameworkBundle\Component\Localization\DisplayTimeZoneProviderInterface|null $displayTimeZoneProvider
+     * @param \Shopsys\FrameworkBundle\Component\Localization\DisplayTimeZoneProvider $displayTimeZoneProvider
```

Comment:
I think it should be annotated with interface too instead of custom implementation

Reply:
true, fixed in 958cbcbe3e, 87d99c5dd4, and c759934ca9